packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -91,13 +98,31 @@ const onRenderParticipantsDefault = (
  * assigned then each participant is rendered with `ParticipantItem`.
  */
 export const ParticipantList = (props: ParticipantListProps): JSX.Element => {
-  const { participants, myUserId, onRenderParticipant, onParticipantRemove, onRenderAvatar } = props;
+  const { excludeMe = false, myUserId, participants, onParticipantRemove, onRenderAvatar, onRenderParticipant } = props;
+
+  const allParticipants: CommunicationParticipant[] = useMemo(() => {
+    if (participants !== undefined) {
```

Comment:
Use early returns to make code flow linear:

    if (participants === undefined) {
        return [];
    }
    if (!(excludeMe && !!myUserId)) {
      return participants;
    }

    const userIndex = participants.map((p) => p.userId).indexOf(myUserId);
    ...
    return remoteParticipants